### DISCUSSION TITLE
GSW Panel

### DISCUSSION THREAD

**Discussion by strayvagrant** [a-sync/game-server-watcher#23] [Q&A] [answered]
How to access the GSW panel?

**Comment by a-sync** [comment]
After starting the app, a web service will start listening on port 8080 by default. To access the panel, enter the address of the server and the port into a web browser.  
So something like: http://localhost:8080 (assuming you are running the service locally)

**Reply by strayvagrant** [reply]
Hi @a-sync, I'm running it on Pterodactyl.

**Reply by a-sync** [reply, marked as answer]
Hi.
Then you need to use the ip (node fqdn) and port that you allocated for your server when you created it.

**Reply by strayvagrant** [reply]
Sup @a-sync, it worked. Thanks for guiding me.
